src/style.css

```
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding-inline-start: 0;
+}
+
+html {
+  --color-nav: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
```

```suggestion
.header {
  display: flex;
  justify-content: space-between;
  padding: 0 50px;
  align-items: center;
}
```

and now you can remove margin-left and margin-top from `logo` and margin-right from `nav`